### PR TITLE
How to create a branch

### PR DESCRIPTION
**Pull request by hemant-sw**
Created a new section for "How to create a new branch" and also "How you can change from an existing branch"

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/open-source-with-pradumna/pull/221"></a>

### CODE REVIEW

File: pages/How-to/guide/create-branch.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## What is Branches in Git ?
+Git Branch is a new or separate version of main repository. A Git project can have more than one branch, branches create another line of deveolpoment in the project. The primary or default branch in Git is the master branch.
```
Comment (AbbeyIT) [inline pages/How-to/guide/create-branch.md:2]: Spelling error for `development`. You spelled it as `deveolpoment`

**Review by AbbeyIT — CHANGES_REQUESTED** [review]

**Comment by AbbeyIT** [comment]
I have a suggestion for this guide. I think it is better to execute the command in the screenshot for better readability and for the users to see the differences between the main branch and when you create a new branch. For example, for using the command `git branch -a`, it is better to show all the available branches in the repo. Because for first-timers, they won't actually get by just saying this command is *to list all of the branches*.

File: pages/How-to/guide/create-branch.md (hunk 1 above)
Comment (Pradumnasaraf) [reply pages/How-to/guide/create-branch.md:2]: Thanks for the suggestions, Abbey

Comment (Pradumnasaraf) [reply pages/How-to/guide/create-branch.md:2]: Thanks for the suggestions, Abbey

**Comment by Pradumnasaraf** [comment]
> I have a suggestion for this guide. I think it is better to execute the command in the screenshot for better readability and for the users to see the differences between the main branch and when you create a new branch. For example, for using the command `git branch -a`, it is better to show all the available branches in the repo. Because for first-timers, they won't actually get by just saying this command is _to list all of the branches_.

I agree with that.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, this issue was primarily focused on how to create a branch with GitHub UI, but it's great you have added Git process. Can you add for UI as well?

**Comment by hemant-sw** [comment]
Hi !! @Pradumnasaraf  as you mentioned I fixed all the changes please have a look. 

thank you

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**